taskcat/_lambda_build.py

```diff
@@ -48,12 +52,70 @@ def _build_lambdas(self, parent_path: Path, output_path):
         if not parent_path.is_dir():
             return
         for path in parent_path.iterdir():
-            if not (path / "Dockerfile").is_file():
-                continue
-            tag = f"taskcat-build-{uuid5(self.NULL_UUID, str(path)).hex}"
-            LOG.info(f"Packaging lambda source from {path} using docker image {tag}")
-            self._docker_build(path, tag)
-            self._docker_extract(tag, output_path / path.stem)
+            if (path / "Dockerfile").is_file():
+                tag = f"taskcat-build-{uuid5(self.NULL_UUID, str(path)).hex}"
+                LOG.info(
+                    f"Packaging lambda source from {path} using docker image {tag}"
+                )
+                self._docker_build(path, tag)
+                self._docker_extract(tag, output_path / path.stem)
+            elif (path / "requirements.txt").is_file():
+                LOG.info(f"Packaging python lambda source from {path} using pip")
+                self._pip_build(path, output_path / path.stem)
+            else:
+                LOG.info(
+                    f"Packaging lambda source from {path} without building "
+                    f"dependencies"
+                )
+                self._zip_dir(path, output_path / path.stem)
+
+    @staticmethod
+    def _make_pip_command(base_path):
```

Comment:
What if pip isn't installed?

Reply:
it will fail, the assumption is that you needed it to install taskcat

Reply:
added pip to requirements